### ISSUE TITLE
Search text

### ISSUE THREAD

**Issue by umerov1999** [zhanghai/MaterialFiles#373]
Please add feture to search text in texts files

**Comment by ioogithub** [comment]
Yes! I have migrated from another file manager and this is the most missed request. In fact I have to keep the old file manage on my phone just because their text file module has a simple hourglass search function at the top of the note which is invaluable for finding strings in config files.
